esp/templates/program/modules/onsitecheckinmodule/checkin.html

```diff
@@ -49,13 +49,16 @@ <h1>Advanced Checkin &mdash; For &quot;{{module.student.name}}&quot;</h1>
 {% else %}
 <div style="margin:10px">
   <p>
-    {{module.student.name}} already checked in at <b>{{module.timeCheckedIn}}</b>.
+    {{module.student.name}} first checked in at <b>{{module.timeCheckedIn}}</b>.
+    {% if module.timeCheckedIn != module.lastCheckedIn %}
+    <br />{{module.student.name}} last checked in at <b>{{module.lastCheckedIn}}</b>.
+    {% endif %}
   </p>
   <form name="checkin" method="POST" action="{{request.path}}">
     <table>
       <tr>
 	<td><label for="attended"><strong>Attending? (i.e. here right now?)</strong></label></td>
-	<td><input type="checkbox" name="attended" id="attended" {% if module.hasAttended %}checked="checked"{% endif %} /></td>
+	<td><input type="checkbox" name="attended" id="attended" {% if module.isAttending %}checked="checked"{% endif %} /></td>
```

Comment:
If you uncheck this box when you don't have the check out module enabled, it seems like it still adds a check out record, instead of deleting the attended record which is what I would imagine should happen when the check out module not enabled?

Reply:
Hmm...I hadn't thought about that. We only want to make checkout records if the checkout module is enabled?

Reply:
Okay nvm- we'll call the checkout records part of the base website and the actual checkout module is only the fancy interface page. So then this is fine that we're making check outs. However-- would we maybe want to allow someone to undo an attended record on this page that they made accidentally? By making a "undo latest attended record for this student" box and a "checkout student" box?